src/content/en/2019/compression.md

```diff
@@ -9,104 +9,104 @@ published: 2019-11-04T12:00:00+00:00:00
 
 Compression algorithms are often categorized as Lossy or Lossless: 
 
-*   When a lossy compression algorithm is used, the process is irreversible and the original file can not be restored via decompression. This is commonly used to compress media resources - such as image and video content. 
+*   When a lossy compression algorithm is used, the process is irreversible, and the original file cannot be restored via decompression. This is commonly used to compress media resources - such as image and video content where losing some data will not materially affect the resource.
 *   Lossless compression is a completely reversible process, and is commonly used to compress text based resources - such as HTML, JavaScript, Stylesheets, etc. 
 
-In this chapter, we are going to explore how text based content is compressed on the web.
+In this chapter, we are going to explore how text-based content is compressed on the web. Analysis of non-text based content forms part of the [Media Chapter](./media).
 
 
-# How HTTP Compression Works
+## How HTTP Compression Works
 
-When a client makes an HTTP request, it often includes an [Accept-Encoding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding) header to advertise the compression algorithms it is capable of decoding. The server can then select from one of the advertised encodings it supports, and serve a compressed response. The compressed response would include a [Content-Encoding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding) header so that the client is aware of which compression was used. Additionally, a [Content-Type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) header is often used to indicate the mime type of the resource being served. 
+When a client makes an HTTP request, it often includes an [Accept-Encoding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding) header to advertise the compression algorithms it is capable of decoding. The server can then select from one of the advertised encodings it supports and serve a compressed response. The compressed response would include a [Content-Encoding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding) header so that the client is aware of which compression was used. Additionally, a [Content-Type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) header is often used to indicate the mime type of the resource being served. 
 
-In the example below, the client advertised support for Gzip, Brotli and Deflate compression. The server decided to return a gzip compressed response containing a text/html document.
+In the example below, the client advertised support for gzip, brotli and deflate compression. The server decided to return a gzip compressed response containing a text/html document.
```

Comment:
Oxford comma

Code format for MIME type value

We may also want to include a link for [brotli](https://en.wikipedia.org/wiki/Brotli) since I assume most people don't know about it.
```suggestion
In the example below, the client advertised support for gzip, brotli, and deflate compression. The server decided to return a gzip compressed response containing a `text/html` document.
```